source/gems/dry-validation/index.html.md

```diff
@@ -16,7 +16,7 @@ sections:
   - error-messages
 ---
 
-Unlike other, well known, validation solutions in Ruby, `dry-validation` takes a different approach and focuses a lot on explicitness, clarity and preciseness of validation logic. It is designed to work with any data input, whether it's a simple hash, an array or a complex object with deeply nested data.
+Unlike other, well known, validation solutions in Ruby, `dry-validation` takes a different approach and focuses a lot on explication, clarity and precision of validation logic. It is designed to work with any data input, whether it's a simple hash, an array or a complex object with deeply nested data.
```

Comment:
Yes. Esp that google translate shows me a different meaning of explication
:)
On Thu, 31 Mar 2016 at 00:11, Tim Riley <notifications@github.com> wrote:

> In source/gems/dry-validation/index.html.md
> <https://github.com/dry-rb/dry-rb.org/pull/48#discussion_r57972883>:
>
> > @@ -16,7 +16,7 @@ sections:
> >    - error-messages
> >  ---
> >
> > -Unlike other, well known, validation solutions in Ruby, `dry-validation` takes a different approach and focuses a lot on explicitness, clarity and preciseness of validation logic. It is designed to work with any data input, whether it's a simple hash, an array or a complex object with deeply nested data.
> > +Unlike other, well known, validation solutions in Ruby, `dry-validation` takes a different approach and focuses a lot on explication, clarity and precision of validation logic. It is designed to work with any data input, whether it's a simple hash, an array or a complex object with deeply nested data.
>
> I like the precision change, but I think explicitness would be better
> understood (and is a real word according to iOS' dictionary).
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/dry-rb/dry-rb.org/pull/48/files/8e22c4af3404e0ba721994e50c0fada0fd3878b6#r57972883>
>